multinode_integration_tests/ci/all.sh

```diff
@@ -14,11 +14,6 @@ export RUST_BACKTRACE=full
 # executable they need will not be in your filesystem but in your (and their) parent's filesystem.  If that's the
 # case, make sure you pass in as parameter 1 the path to the directory just above the 'node' module directory,
 # IN THE CONTEXT OF THE PARENT (host) FILESYSTEM.
```

Comment:
You could get rid of these comments too. They don't comment on anything anymore.